lib/Onyx.js

```diff
@@ -199,9 +203,11 @@ function addToEvictionBlockList(key, connectionID) {
  * the recently accessed list when initializing the app. This
  * enables keys that have not recently been accessed to be
  * removed.
+ *
+ * @returns {Promise}
  */
 function addAllSafeEvictionKeysToRecentlyAccessedList() {
-    getAllKeys()
+    return getAllKeys()
```

Comment:
return the Promise here so we can wait for it as well
Since every `connect` uses `getAllKeys` it would be best to wait for the read here before any connections are made
Then every connection would instantly get all keys from cache